pyproj/_crs.pyx

```diff
@@ -489,6 +504,38 @@ cdef class Ellipsoid(Base):
         """
         return Ellipsoid.from_authority("EPSG", code)
 
+    @staticmethod
+    def from_string(ellipsoid_string):
+        """
+        Create an Ellipsoid from a string.
+
+        Examples:
+          - urn:ogc:def:ellipsoid:EPSG::7001
+          - ELLIPSOID["Airy 1830",6377563.396,299.3249646,
+                LENGTHUNIT["metre",1],
+                ID["EPSG",7001]]
+        Parameters
```

Comment:
Blank line needed between the example and the Parameters (rst / sphinx is quit picky about white space)

Reply:
Good catch.

Reply:
I tested with sphinx and needed to make some other fixes, but all should be good now. Thanks!